review-me/review-me.js

```diff
@@ -0,0 +1,31 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
+    }
+}
+
+printmessage(message)
```

Comment:
15  :  Use  of  eval  Security  risk  and  code  smell ;  should  never  use  eval